src/modules/iotjs_module_spi.c

```diff
@@ -174,12 +174,12 @@ static void iotjs_spi_release_buffer(iotjs_spi_t* spi) {
 
 
 static void iotjs_spi_set_configuration(iotjs_spi_t* spi,
-                                        const iotjs_jval_t* joptions) {
+                                        iotjs_jval_t joptions) {
```

Comment:
Please change `joptions` to `&joptions` at 186 line.

Reply:
Good catch. Fixed.

Reply:
@hs0225 why SPI module is not included in atrik053 precommit test? Is it intended?

Reply:
@LaszloLango No. I'll fix it.